[2023-02-10T23:07:55.796+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T09:20:00+00:00 [queued]>
[2023-02-10T23:07:55.802+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T09:20:00+00:00 [queued]>
[2023-02-10T23:07:55.802+0200] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:07:55.802+0200] {taskinstance.py:1369} INFO - Starting attempt 1 of 4
[2023-02-10T23:07:55.802+0200] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:07:55.807+0200] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data_from_api> on 2023-01-10 09:20:00+00:00
[2023-02-10T23:07:55.811+0200] {standard_task_runner.py:52} INFO - Started process 16644 to run task
[2023-02-10T23:07:55.814+0200] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_process_sales', 'extract_data_from_api', 'scheduled__2023-01-10T09:20:00+00:00', '--job-id', '941', '--raw', '--subdir', 'DAGS_FOLDER/process_sales/mainS.py', '--cfg-path', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpto3rrse2', '--error-file', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpoirh9zuh']
[2023-02-10T23:07:55.815+0200] {standard_task_runner.py:80} INFO - Job 941: Subtask extract_data_from_api
[2023-02-10T23:07:55.841+0200] {task_command.py:371} INFO - Running <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T09:20:00+00:00 [running]> on host admins-MacBook-Air-2.local
[2023-02-10T23:07:55.867+0200] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_process_sales
AIRFLOW_CTX_TASK_ID=extract_data_from_api
AIRFLOW_CTX_EXECUTION_DATE=2023-01-10T09:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-10T09:20:00+00:00
[2023-02-10T23:07:56.170+0200] {python.py:173} INFO - Done. Returned value was: {'message': 'Authorization token not set properly. You are not authorised to use this resourse'}
[2023-02-10T23:07:56.184+0200] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_process_sales, task_id=extract_data_from_api, execution_date=20230110T092000, start_date=20230210T210755, end_date=20230210T210756
[2023-02-10T23:07:56.231+0200] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10T23:07:56.242+0200] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
